src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+export default function ArticlePage() {
+    const [article, setArticle] = useState({});
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        axios
```

Comment:
I strongly recommend extracting api calls into a utils file of functions as this will really tidy up your components and make them easier to work with and more readable

Reply:
Not entirely sure how to do this, initially I set this up but couldn't get it to work.

Reply:
Hiya Chris, if you're not sure how to do this you can have a look at these notes https://notes.northcoders.com/courses/js-front-end/axios#using-axios-to-make-http-requests
Here they've separated out the API request into a function (in your case you'd separate this out into a separate api.js file and import it.
You don't have to do this and you can merge without doing it, but it will make things easier going forward to have all your api calls in one location.